packages/seacas/libraries/ioss/src/main/io_shell_ts.C

```diff
@@ -177,25 +205,6 @@ int main(int argc, char *argv[])
 
 #ifdef SEACAS_HAVE_KOKKOS
   Kokkos::initialize(argc, argv);
```

Comment:
Use `Kokkos::ScopeGuard` instead of `Kokkos::initialize` + `Kokkos::finalize`.

Reply:
Thanks, I will change that.

Reply:
@gsjaardema I think you may have misunderstood -- Kokkos already has the `Kokkos::ScopeGuard` class.  Also, Teuchos has `Teuchos::GlobalMPISession` for MPI.